### PR TITLE
Auth

### LINKED ISSUE #13: Security
Make it so you have to log in to do anything. Meaning no uploading or viewing if you aren't authenticated. Redirect user to login page. Update Nav to include all the correct items, /upload and /gallery.

### PR DESCRIPTION
**Pull request by jaw187**
Closes #13

### CODE REVIEW

File: hapchat/index.js [hunk 1]
```diff
@@ -66,10 +65,39 @@ internals.main = function main() {
...
+        Bell
+    ], function (err) {
+
+        // Set session
+        server.auth.strategy('session', 'cookie', {
+            password: 'cookie_encryption_password',
+            cookie: 'sid',
+            isSecure: false,
+            redirectTo: '/login',
+            ttl: 24 * 60 * 60 * 1000                          // 1 Day
```
Comment (arb) [inline hapchat/index.js]: Don't we have `one_day` already attached to server?

**Comment by arb** [comment]
Doesn't appear to work.
screen shot 2014-10-17 at 9 50 04 am

**Comment by arb** [comment]
NICE! Seems to be working. Can you attach the "user name" or whatever we have on hand to the global context so we can so like "hi @jaw187". You don't need to do the UI, just attach the data.

**Comment by arb** [comment]
So for now, let's just whack the Facebook login stuff to keep the code size down. We can add more down the road if we want. Also, should the login page have like an icon on it now for like "Log in with GitHub" rather than going to /login automatically taking you to The GitHub?

**Comment by jaw187** [comment]
All of the user data can be found in `request.auth.session.profile`

**Comment by jaw187** [comment]
Feel free to change the login page how ever you see fit.  For auth purposes I hope this PR is good to go.

**Pull request merged**